CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # Release Notes
 
+## Nerves v0.9.0
+
+* Update Notes
+
+Starting in Nerves v0.9.0 artifacts will no longer be fetched during
+`mix compile`. Artifact archives are intended to be fetched following `mix deps.get`
+To handle this, you will need to update your installed version of `nerves_bootstrap` by
+callin `mix nerves.local`. After updating `nerves_bootstrap`, you should update
+your mix file to include calls to the new required Nerves mix aliases by replacing
```

Comment:
I think the way you've structured this is a little confusing since it requires reading code as well as english while trying to understand what changes are being asked of me at the same time.  I would fully state in English what is being asked first, and then show an example.  For example, you could say:

>After updating `nerves_bootstrap`, you should update your mix.exs file to add the new required mix aliases found there.  A helper function is available named `Nerves.Bootstrap.add_aliases` that you can pipe your existing aliases to like this: